tasks/eth/4-protocol-versions/README.md

```diff
@@ -0,0 +1,204 @@
+
+```
+cd superchain-ops
+git pull
+just install
+cd tasks/eth/4-protocol-version
+```
+
+### 2. Setup Ledger
+
+Your Ledger needs to be connected and unlocked. The Ethereum
+application needs to be opened on Ledger with the message "Application
+is ready".
+
+### 3. Simulate and validate the transaction
+
+Make sure your ledger is still unlocked and run the following.
+
+Remember that by default `just` is running with the address derived from
+`/0` (first nonce). If you wish to use a different account, run `just
+simulate [X]`, where X is the derivation path of the address
+that you want to use.
+
+You will see a "Simulation link" from the output.
+
+Paste this URL in your browser. A prompt may ask you to choose a
+project, any project will do. You can create one if necessary.
+
+Click "Simulate Transaction".
+
+We will be performing 3 validations and extract the domain hash and
+message hash to approve on your Ledger:
+
+1. Validate integrity of the simulation.
+2. Validate correctness of the state diff.
+3. Validate and extract domain hash and message hash to approve.
+
+#### 3.1. Validate integrity of the simulation.
+
+Make sure you are on the "Overview" tab of the tenderly simulation, to
+validate integrity of the simulation, we need to check the following:
+
+1. "Network": Check the network is mainnet.
+2. "Timestamp": Check the simulation is performed on a block with a
+   recent timestamp (i.e. close to when you run the script).
+3. "Sender": Check the address shown is your signer account. If not,
+   you will need to determine which “number” it is in the list of
+   addresses on your ledger. By default the script will assume the
+   derivation path is `m/44'/60'/0'/0/0`. By calling the script with
+   `just simulate 1` it will derive the address using
+   `m/44'/60'/1'/0/0` instead.
+
+#### 3.2. Validate correctness of the state diff.
+
+Now click on the "State" tab. Verify that:
+
+* For the `ProtocolVersions` Proxy at `0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935` both the
+  recommended and required storage slots are updated from the encoded form of `3.1.0` to `5.0.0`.
+  * key `0x4aaefe95bd84fd3f32700cf3b7566bc944b73138e41958b5785826df2aecace0`
+    * before: `0x0000000000000000000000000000000000000003000000010000000000000000`
+    * after : `0x0000000000000000000000000000000000000005000000000000000000000000`
+  * key `0xe314dfc40f0025322aacc0ba8ef420b62fb3b702cf01e0cdf3d829117ac2ff1a`
+    * before: `0x0000000000000000000000000000000000000003000000010000000000000000`
+    * after : `0x0000000000000000000000000000000000000005000000000000000000000000`
+* All other state changes (2) are a nonce change of the sender account and the multisig.
+
+On the "Events" tab, you can verify that two `ConfigUpdate` events were emitted from the `ProtocolVersions` proxy,
+as well as an `ExecutionSuccess` event by the multisig.
+
+![](./images/tenderly-state.png)
+
+You can verify the correctness of the storage slots with `chisel`.
```

Comment:
I cannot reliably run `chisel`. It fails with this error when I run it in this repo:
```
solc 0.8.15 is not supported by the set evm version: paris. Please install and use a version of solc higher or equal to 0.8.18.
You can also set the solc version in your foundry.toml.
```

It's also not a tool that we've asked signers to use before, so I anticipate confusion. 

Could we instead suggest using:

```
cast keccak "protocolversion.required"
```

And indicate that the slot changed should be one less then what is returned by that?

Reply:
Does your `chisel` installation run normally in other repos? I just use the latest installation from `foundryup`, which is also recommended in the README of this repo. 

I'm wondering if asking signers to subtract `1` from a `cast` call might be more confusing than asking them to copy `bytes32(uint256(keccak256("protocolversion.required")) - 1)` into `chisel`.

But if you still think it's better with `cast keccak`, we will adapt this section.

Reply:
I can also completely remove this section on storage slot verification, if that's not required anyways. I found it a nice additional sanity check.

Reply:
I see that you've included both which LGTM